amd_openvx/cmake/FindTurboJpeg.cmake

```diff
@@ -25,21 +25,30 @@
 ################################################################################
 find_path(TurboJpeg_INCLUDE_DIRS
     NAMES turbojpeg.h
+    HINTS
+    $ENV{TURBO_JPEG_PATH}/include
     PATHS
+    ${TURBO_JPEG_PATH}/include
     /usr/include/
 )
 mark_as_advanced( TurboJpeg_INCLUDE_DIRS )
 
 find_library( TurboJpeg_LIBRARIES
     NAMES libturbojpeg.so
+    HINTS
+    $ENV{TURBO_JPEG_PATH}/lib
     PATHS
+    ${TURBO_JPEG_PATH}/include
```

Comment:
${TURBO_JPEG_PATH}/lib

Reply:
also, it would be good to add lib64 too...as when i tried with this change , i found the libjpegturbo.so was installed in /lib64  as in the case below 
./opt/spack/linux-centos8-skylake_avx512/gcc-8.4.1/libjpeg-turbo-2.1.3-y5dxc36owqbi5kxgk6jih7543nbmvbkw/lib64/libturbojpeg.so

Reply:
@srekolam added support for lib64